From da51d10371034bbb577652d13b25b332f982a4dc Mon Sep 17 00:00:00 2001
From: Andreas Oberritter <obi@linuxtv.org>
Date: Tue, 12 Oct 2010 10:03:53 +0000
Subject: [PATCH 1/4] make_makefile.pl: use $(CROSS_COMPILE)strip

---
 v4l/scripts/make_makefile.pl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/v4l/scripts/make_makefile.pl b/v4l/scripts/make_makefile.pl
index 6f51209..8d99e4e 100755
--- a/v4l/scripts/make_makefile.pl
+++ b/v4l/scripts/make_makefile.pl
@@ -254,7 +254,7 @@ while (my ($dir, $files) = each %instdir) {
 	print OUT "echo -n \"\$\$i \"; ";
 	print OUT "install -m 644 -c \$\$i \$(DESTDIR)\$(KDIR26)/$dir; fi; done; ";
 	print OUT "if [  \$\$n -ne 0 ]; then echo; ";
-	print OUT "strip --strip-debug \$(DESTDIR)\$(KDIR26)/$dir/*.ko; ";
+	print OUT "\$(CROSS_COMPILE)strip --strip-debug \$(DESTDIR)\$(KDIR26)/$dir/*.ko; ";
 	print OUT "fi;\n\n";
 }
 print OUT "\t@echo\n";
-- 
2.25.1

